src/Build/Microsoft.Build.csproj

```diff
@@ -20,6 +20,8 @@
     <PackageDescription>This package contains the $(MSBuildProjectName) assembly which is used to create, edit, and evaluate MSBuild projects.</PackageDescription>
     <IncludeSatelliteOutputInPack>false</IncludeSatelliteOutputInPack>
     <ApplyNgenOptimization Condition="'$(TargetFramework)' == '$(FullFrameworkTFM)'">full</ApplyNgenOptimization>
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <PackageValidationBaselineVersion>17.3.2</PackageValidationBaselineVersion>
```

Comment:
Do the packages for which you enabled package validation, version independently? If not, it would make sense to centralize the version, i.e. in the Versions.props file or somewhere else.

Reply:
They used to, but they shouldn't anymore. Thanks!